lib/pbench/server/api/resources/users_api.py

```diff
@@ -59,15 +59,22 @@ def post(self):
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
 
         if not user:
-            self.logger.exception("No such user exists with the given email ID")
-            abort(HTTPStatus.BAD_REQUEST, message="No such user exists with the given email ID")
+            self.logger.warning("No such user exists with the given email ID")
+            return "", HTTPStatus.OK
+
+        # generate auth token for the valid user
+        token = self.auth.encode_auth_token("60", user.id)
 
         # generate email and send to the user
-        msg = Message(subject="Hello World!",
+        # Ideally the message should be a template
+        # rendered using render_template
+        msg = Message(subject="Reset your password!",
                       sender=self.config.get("pbench-server", "email_username"),
                       recipients=[email],
-                      body="Hello World"
-        )
+                      body=("Please reset your password at http://localhost:8000/reset_password/" + token)
```

Comment:
`http://localhost:8000` is just a "stub", right?  I presume that the URL is available via `self.config.get()`.

Reply:
That's right